kedro/io/kedro_data_catalog.py

```diff
@@ -277,7 +355,35 @@ def save(self, name: str, data: Any) -> None:
         dataset.save(data)
 
     def load(self, name: str, version: str | None = None) -> Any:
-        """Loads a registered dataset."""
+        # TODO: rename input argument when breaking change: name -> ds_name
```

Comment:
Same as for save, let's keep it `name`.

Reply:
We agreed to rename it for consistency with arguments for other methods.